server/feeds/examples/feed-cursor.yaml

```diff
@@ -30,8 +31,8 @@ components:
             id: 78d98a6a-c24b-4b7a-8e06-4a30862afa2d
             parent_board_slug: long
             parent_board_id: db8dc5b3-5b4a-4bfe-a303-e176c9b00b83
-            parent_realm_slug: 'twisted-minds'
-            parent_realm_id: '76ef4cc3-1603-4278-95d7-99c59f481d2e'
+            parent_realm_slug: "twisted-minds"
```

Comment:
are the double quotes vs single quotes in yaml files a personal preference thing or a "recommended syntax" thing?

Reply:
This is how prettier "fixed" them when i saved the file, and theoretically we just abide to prettier's whims when it comes to formatting.